IDEAS/Buildings/Components/OccupancyType/OfficeWork.mo

```diff
@@ -0,0 +1,17 @@
+within IDEAS.Buildings.Components.OccupancyType;
+record OfficeWork
+  "Properties for typical office work. Based on Fanger and Ashrae Fundamentals"
```

Comment:
Can you refer to actual ASHRAE number ? 
This description is actually too long: it shows in the dropdown and then you just see "properties of typical ..."